projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -83,6 +83,16 @@ function get_plan_selected_subtitle() {
 		) : '';
 }
 
+/**
+ * Determines whether or not design selected task is enabled
+ *
+ * @return boolean True if design selected task is enabled
+ */
+function can_update_design_selected_task() {
+	$site_intent = get_option( 'site_intent' );
+	return $site_intent === 'newsletter' || $site_intent === 'build' || $site_intent === 'write';
+}
```

Comment:
Should the function above be checking for 'free' rather than 'newsletter'? I think we're trying to duplicate this front end logic? 

https://github.com/Automattic/wp-calypso/blob/fdaa3105365adae2597ef4de6052655c1b811522/client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts#L47

Reply:
Thanks for catching that 😅

Reply:
You're totally right

Reply:
Updated in [0e609d3](https://github.com/Automattic/jetpack/pull/30116/commits/0e609d35a28e688f655b764089d8500927481020)